typedb-src/modules/ROOT/pages/overview.adoc

```diff
@@ -2,25 +2,25 @@
 :keywords: typedb, typeql, documentation, overview, introduction
 :longTailKeywords: documentation overview, learn typedb, learn typeql, typedb schema, typedb data model
 :pageTitle: Documentation overview
-:summary: A birds-eye view of TypeQL and TypeDB
+:summary: A bird's-eye view of TypeQL and TypeDB
```

Comment:
```suggestion
:summary: Overview of the TypeDB documentation
```
1. We now have a dedicated TypeQL section.
2. I suggest we adopt this approach to naming overview pages in all sections.

Reply:
The "the" (definite article) here is nonstandard.

Reply:
Replaced with the phrase that won the vote in https://github.com/vaticle/typedb-docs/pull/690/commits/eb32b4522c34d38a5c059c77636c45d0d32a7374